packages/account/src/providers/transaction-summary/input.test.ts

```diff
@@ -120,4 +121,18 @@ describe('transaction-summary/input', () => {
       MOCK_INPUT_MESSAGE
     );
   });
+
+  it('should ensure getInputFromAssetId returns input message if the coin input amount is 0', () => {
```

Comment:
I don't believe this description is describing the behaviour that is actually happening, as the reason the message input is selected is because the `assetId` of `MOCK_INPUT_COIN` does not equal the `ASSET_A`.

Reply:
[`63e827d` (#2782)](https://github.com/FuelLabs/fuels-ts/pull/2782/commits/63e827df7cf85317ae9a9017edbe7d22fe54c845)